lean4-infoview/src/infoview/rpcInterface.ts

```diff
@@ -152,3 +154,13 @@ export async function InteractiveDiagnostics_msgToInteractive(rs: RpcSessions, p
     if (ret) TaggedMsg_registerRefs(rs, pos, ret)
     return ret
 }
+
+export type GoToKind = 'definition' | 'declaration' | 'type'
```

Comment:
Your requesthandling.lean defines GoToKind in a different order `| declaration | definition | type`